configure.ac

```diff
@@ -1692,8 +1693,20 @@ AS_IF([test "x$enable_tsan" = "xyes" ],
           [gcc-*|clang-*],
           [tsan=true],
           [AC_MSG_ERROR([thread sanitizer not supported with compiler \
-$cc_basename"])]
+$ocaml_cv_cc_vendor"])]
          )],
+        [AC_MSG_ERROR([thread sanitizer not supported on system $system])]
+       )],
+    [arm64],
+      [AS_CASE(["$system"],
+        [linux],
+        [AS_CASE(["ocaml_cv_cc_vendorcc_basename"],
```

Comment:
copy-paste typo?

Reply:
Oops. Last-minute edit botch under vi, sorry. Thanks for noticing!